src/main/java/org/example/service/JsonResponseService.java

```diff
@@ -0,0 +1,48 @@
+package org.example.service;
+
+import java.io.PrintWriter;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+public class JsonResponseService {
+
+    private final ConnectionService connectionService = ConnectionService.getInstance();
+
+    private static final String NAME_OF_TABLE = "developer";
+
+    private static JsonResponseService INSTANCE = null;
+
+    public String jsonSender() {
```

Comment:
Pass the developers as an argument for this method.
And get the JSON output of the developers.

Reply:
fixed but i used something more complicated that you say

Reply:
Yeah, I have checked... but why you can not pass the developers as an argument for this method?

Reply:
fixed